metadata-ingestion/src/datahub/ingestion/source/usage/usage_common.py

```diff
@@ -213,19 +213,6 @@ def ensure_top_n_queries_is_not_too_big(cls, v: int) -> int:
             )
         return v
 
-    @pydantic.validator("start_time")
```

Comment:
this is removed to allow user to specify exact absolute start_time if necessary for some reason. In case of scheduled ingestions (default/relative start time), aligning start_time with bucket start time is already taken care of elsewhere.